indigo/distribution.yaml

```diff
@@ -752,6 +752,13 @@ repositories:
       type: git
       url: https://github.com/astuff/ros_pacmod.git
       version: release
+    release:
+      packages:
+      - pacmod
```

Comment:
The naming starts getting confusing here:
- the repository entry is `astuff_pacmod`
- the actual repository name is `ros_pacmod`
- the package hosted on that repository is `pacmod`

Not a blocker for merging this but it would benefit from being homogenized IMO

Reply:
I'm going to go ahead and close this PR and squash everything down to just pacmod before continuing. Please don't merge.

Reply:
As this seem to be the pattern followed by the other astuff repositories I'll go ahead and merge this as I expect homogenizing the pattern would be disruptive :+1:

Reply:
sorry we wrote at the same time, if this is possible on your end that's great !